### ISSUE TITLE
Add a tab to the website for searching by gene 

### ISSUE THREAD

**Issue by NathanSkene** [neurogenomics/rare_disease_celltyping_apps#16]
Should be able to search for a particular gene, see where it is expressed, then find phenotypes and cell types associated with it, and what its expression is across the cell types

It should also be possible to click through to this, from e.g. the phenotypes tab, when you see what genes underlie a phenotype

The tab should also link through to the gnomAD browser page for that gene: https://gnomad.broadinstitute.org/gene/ENSG00000187498?dataset=gnomad_r4

The tab should also link through to the MGI page:
https://www.informatics.jax.org/marker/MGI:88454

In the future, we will want to expand on this...
- E.g. allow filtering of genes which are associated with severe phenotypes, have matching MGI phenotypes, have Loss-of-Function variants within them which are not associated with pathogenicity etc.
